src/general/semidiscretization.jl

```diff
@@ -788,6 +788,18 @@ function update_nhs!(neighborhood_search,
     return neighborhood_search
 end
 
+# Forward to PointNeighbors.jl
+function update!(neighborhood_search, system, x, y; points_moving=(true, false))
+    PointNeighbors.update!(neighborhood_search, x, y; points_moving)
+end
+
+# For `GPUSystem`s, explicitly pass the backend, so that with a `GPUSystem` with CPU
+# backend, it will actually launch the KernelAbstractions.jl kernels on the CPU.
```

Comment:
```suggestion
# For `GPUSystem`s, explicitly pass the backend, so that a `GPUSystem` with a CPU
# backend will actually launch the KernelAbstractions.jl kernels on the CPU.
```